### ISSUE TITLE
Doesn't find a starspot solution

### ISSUE THREAD

**Issue by therob762** [bkloppenborg/simtoi#135]
I have attached the .json file of the 256x256 model I was running on the attached data file.  In short, this is a single starspot on the surface of a rapid rotator where multinest is run to find the polar radius, the power-law limb darkening coefficient, and the four starspot parameters.  This is the output when I first start the model:
```
"X Error: BadWindow (invalid Window parameter) 3
  Major opcode: 20 (X_GetProperty)
  Resource id:  0x7c0004b
Data set information for: 
 /home/parksj/research/data/inter/sig_per/sig_Per.oifits
N Vis: 0
N V2 : 216
N T3 : 272
N UV : 1028
Average JD: 2455828.96234
X Error: BadWindow (invalid Window parameter) 3
  Major opcode: 20 (X_GetProperty)
  Resource id:  0x7c00051
 *****************************************************
 MultiNest v3.9
 Copyright Farhan Feroz & Mike Hobson
 Release Oct 2014

 no. of live points =  200
 dimensionality =    6
 *****************************************************
 Starting MultiNest
 generating live points"
```
I let the code run for over 12 hours and the only output that was generated is when I exit SIMTOI.  This is the output upon exit:
```
"Exit signal received
Could not read multinestsummary.txt file.
Best-fit parameters and exported values will be invalid.
terminate called after throwing an instance of 'std::range_error'
  what():  The new value for R_pole is less than the set minimum.
Aborted (core dumped)"
```
Multinest seemed to be running the entire time, however just prior to exiting it didn't seem any closer to the solution than when it began.

Okay, so I can't attach the .json file, but here is the text:
```
{
	"_file_info" : "Save file from the SImulation and Modeling Tool for Optical Interferometry (SIMTOI) version 1.1.0 commit 069c52f. See https://github.com/bkloppenborg/simtoi for information about this format",
	"area_height" : 256,
	"area_scale" : 0.050000000000000003,
	"area_width" : 256,
	"model_0" : 
	{
		"base_data" : 
		{
			"T_eff_pole" : [ 5000, 2000, 1000000, false, 100, 0 ],
			"g_pole" : [ 1, 0.01, 10000, false, 10, 2 ],
			"inclination" : [ 0, -180, 180, false, 1, 2 ],
			"n_side_power" : [ 4, 1, 10, false, 1, 0 ],
			"omega_rot" : [ 0.5, 0, 1, false, 0.10000000000000001, 2 ],
			"position_angle" : [ 0, 0, 360, false, 0.10000000000000001, 2 ],
			"r_pole" : [ 3, 2.5, 3.5, true, 1, 4 ],
			"von_zeipel_beta" : [ 0.5, 0.01, 1, false, 0.10000000000000001, 2 ],
			"z_axis_rotation" : [ 0, 0, 360, false, 1, 2 ],
			"z_axis_rotational_period" : [ 0, 0, 100, false, 1, 4 ]
		},
		"base_id" : "roche_rotator",
		"feature_0_data" : 
		{
			"delta_T" : [ -500, -2000, 0, true, 100, 0 ],
			"phi" : [ 0, 0, 180, true, 0.10000000000000001, 2 ],
			"radius" : [ 1, 0, 10, true, 1, 2 ],
			"theta" : [ 0, 0, 180, true, 0.10000000000000001, 2 ]
		},
		"feature_0_id" : "uniform_spot",
		"position_data" : 
		{
			"E" : [ 0, -1, 1, false, 0.10000000000000001, 2 ],
			"N" : [ 0, -1, 1, false, 0.10000000000000001, 2 ]
		},
		"position_id" : "xy",
		"shader_data" : 
		{
			"alpha" : [ 0.25, 0.10000000000000001, 0.5, true, 0.089999999999999997, 1 ]
		},
		"shader_id" : "ldl_power_law"
	}
}
```

**Comment by fabienbaron** [comment]
To make things clearer, Multinest doesn't seem to be able to generate all the live points, so it never gets to the minimization part.

**Comment by bkloppenborg** [comment]
After @therob762 and I investigated this issue more, we found the `v2err` field in the data file contained `-1` values. This resulted in SIMTOI sending `NAN` values to MultiNest which, in turn, caused the solution not to progress. In fbbc25effa5abf0466cbfa58b2749e63c197df13 I pulled in an updated version of `ccoifits` which tests for `-1` in the error field and (silently) sets flags in the error column. This is now fixed in the develop branch. Be sure to do a `git pull` and `git submodule update` to get the revised version of `ccoifits`
